Meadowlark-js/packages/meadowlark-core/src/Logger.ts

```diff
@@ -6,20 +6,44 @@
 import winston from 'winston';
 import { FrontendRequest } from './handler/FrontendRequest';
 
-const offline = process.env.IS_LOCAL === 'true';
+const timestampFormat: string = 'YYYY-MM-DD HH:mm:SS';
+
+const convertErrorToString = (err) => {
+  // Preserve any dictionary, but otherwise convert to string
+  if (err != null && err.constructor !== Object) {
+    return err.toString();
+  }
+  return err;
+};
 
 const format = winston.format.combine(
   winston.format.label({
     label: 'Meadowlark',
   }),
   winston.format.timestamp({
-    format: 'YYYY-MM-DD HH:mm:SS',
+    format: timestampFormat,
   }),
   winston.format.json(),
 );
 
 const offlineFormat = winston.format.combine(
-  winston.format.cli(),
+  winston.format.timestamp({
+    format: timestampFormat,
+  }),
+  winston.format.printf(({ level, message, timestamp, extra }) => {
+    let m = message;
+    let e = extra ?? '';
+
+    if (typeof message === 'object') {
+      // TypeScript thinks that this is a string, but it there are cases where it ends up being an object
+      m = convertErrorToString(m);
+    }
+    if (typeof extra === 'object') {
+      e = JSON.stringify(extra);
+    }
+
+    return `${timestamp} ${level} ${m} ${e}`;
```

Comment:
Now that I've fixed `IS_LOCAL`, the `winston.format.cli()` formatter will ignore the `extra` stuff. This configuration has output like this:

```
2022-07-26 17:29:76 info Server listening at http://127.0.0.1:3000
2022-07-26 17:29:77 info Starting Meadowlark API at http://127.0.0.1:3000/local
2022-07-26 17:29:67 info Request {"url":"/local/v3.3b/ed-fi/contentClassDescriptors}
```